src/main/java/gregtech/integration/jei/recipe/GTRecipeWrapper.java

```diff
@@ -111,11 +111,37 @@ public void drawInfo(Minecraft minecraft, int recipeWidth, int recipeHeight, int
         minecraft.fontRenderer.drawString(I18n.format("gregtech.recipe.total", Math.abs((long) recipe.getEUt()) * recipe.getDuration()), 0, 70, 0x111111);
         minecraft.fontRenderer.drawString(I18n.format(recipe.getEUt() >= 0 ? "gregtech.recipe.eu" : "gregtech.recipe.eu_inverted", Math.abs(recipe.getEUt())), 0, 80, 0x111111);
         minecraft.fontRenderer.drawString(I18n.format("gregtech.recipe.duration", recipe.getDuration() / 20f), 0, 90, 0x111111);
-        int baseYPosition = 100;
+        minecraft.fontRenderer.drawString(I18n.format("gregtech.recipe.min_tier", getMinTierForVoltage(recipe.getEUt())), 0, 100, 0x111111);
+
+        int baseYPosition = 110;
         for (String propertyKey : recipe.getPropertyKeys()) {
             minecraft.fontRenderer.drawString(I18n.format("gregtech.recipe." + propertyKey,
                 recipe.<Object>getProperty(propertyKey)), 0, baseYPosition, 0x111111);
             baseYPosition += 10;
         }
     }
+
+    private String getMinTierForVoltage(long voltage) {
```

Comment:
This function is redundant and wrong (using magic numbers and hardcoded strings is not good practice). Please use functionality from https://github.com/GregTechCE/GregTech/blob/master/src/main/java/gregtech/api/util/GTUtility.java#L380 and https://github.com/GregTechCE/GregTech/blob/master/src/main/java/gregtech/api/GTValues.java#L58 instead to get correct tier name.

Reply:
It looks like GTUtility.getTierByVoltage is returning values off by 1, when referenced against VN
eg GTValues.VN[GTUtility.getTierByVoltage(recipe.getEUt())] for "Nicken Zinc Ferrite Dust" at 60 EU/t should be MV, but this returns LV.
Is this to be expected?

Reply:
https://gist.github.com/clienthax/594dda020925b92e61742a80c70a6912

Reply:
Interesting I did not expect that. It seems that function I recommend is not suited for this job. But that does not make much difference. Please create similar function in integration/jei/utils which will correctly calculate minimal tier using constants from GTValues.V and GTValues.VN and use that instead.

Reply:
Yeah that function in GTUtility is odd, I would say its either badly named or just not returning what its meant to.